app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -269,15 +269,27 @@ export default {
             weight: 40,
             components: {
               // Need to re-define Formio basic fields here
-              textfield: true,
-              textarea: true,
-              number: true,
-              password: true,
-              checkbox: true,
-              selectboxes: true,
-              select: true,
-              radio: true,
-              button: true,
+              // To disable fields make it false here
```

Comment:
No, it will not create any problems for the older forms. Because we are just removing these default components from the advanced fields section and then adding them with little customization on the index [here](https://github.com/bcgov/common-hosted-form-service/blob/final-mail-period-notification/components/src/components/index.ts)

So here is a bit more info about this. As you know that using a Default component will not help to modify a component, As per requirement we need to add a new checkbox "Allow value propagation" under the validation tab so that a user can choose fields that can be used for _Copy from existing submission_ feature. To do so, I have used Form.io's feature to extend an existing component and add modifications to fulfill our requirements. For example, I have disabled Form io's default TextField for advanced fields and then create a new TextField component by extending from the default one [here](https://github.com/bcgov/common-hosted-form-service/tree/final-mail-period-notification/components/src/components/SimpleTextFieldAdvanced) and used as a component. So even if an old form has this Component used in the schema, Will adapt this new extended component.

I hope I answered it well but still if I miss something please feel free to shoot me a message :)